lib/exposes.js

```diff
@@ -643,6 +643,7 @@ module.exports = {
         valve_detection: () => new Switch().withState('valve_detection', true).setAccess('state', access.STATE_SET),
         vibration: () => new Binary('vibration', access.STATE, true, false).withDescription('Indicates whether the device detected vibration'),
         voc: () => new Numeric('voc', access.STATE).withUnit('ppb').withDescription('Measured VOC value'),
+        voc_index: () => new Numeric('voc_index', access.STATE).withDescription('Sensirion VOC index'),
```

Comment:
Do we know if this measurement is also ppb? If it's better to reuse the above voc() exposes to keep things consistent.

Reply:
It's a unitless, relative property invented by Sensirion. I wrote a bit about it [here](https://github.com/Koenkk/zigbee2mqtt/issues/16717#issuecomment-1436874652). There's also a short [summary PDF](https://sensirion.com/media/documents/02232963/6294E043/Info_Note_VOC_Index.pdf) from the company itself about it.

Reply:
Interesting, makes me wonder if we got the Develco one wrong also, I got my hands on this when adding support for it via their support e-mail.

[Sensirion_Gas_Sensors_SGP3x_TVOC_Concept.pdf](https://github.com/Koenkk/zigbee-herdsman-converters/files/10888459/Sensirion_Gas_Sensors_SGP3x_TVOC_Concept.pdf)

But it wouldn't surprise me that the values it's spewing is not ppb either, but it at least does not seem to be a relative value like you're describing IKEA is using.

Value range from 0 -> ~30k

<img width="1760" alt="image" src="https://user-images.githubusercontent.com/379665/222905514-47d51733-8330-4d25-a9f4-d1b210d1df65.png">

So indeed separating them makes sense here.

Reply:
So, in contrast to the SEN54 the Vindstyrka is using, the SGP30 appears to actually output tVOC in ppb.